src/background/background.ts

```
@@ -79,34 +77,32 @@ chrome.runtime.onMessage.addListener(async (request, sender, sendResponse) => {
-                const subscriptionsRaw =
-                    request.value.userClaim.config.subscriptions;
-                const subscriptionsMap = Object.keys(subscriptionsRaw).reduce(
-                    (acc, key) => {
-                        return {
-                            ...acc,
-                            [key]: subscriptionsRaw[key].value === "true",
-                        };
-                    },
-                    {} as Record<string, boolean>
-                );
+            const subscriptionsRaw = request.value.userClaim.config.subscriptions || {};
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)